src/main/java/com/adyen/service/legalentitymanagement/Documents.java

```diff
@@ -4,6 +4,7 @@
 import com.adyen.Service;
 import com.adyen.constants.ApiConstants;
 import com.adyen.model.legalEntityManagement.Document;
+import com.adyen.model.legalEntityManagement.JSON;
```

Comment:
Could you also please rename the models package to `com.adyen.model.legalentitymanagement`?

Reply:
done!